azurerm/resource_arm_media_services_account.go

```diff
@@ -136,7 +135,10 @@ func resourceArmMediaServicesAccountRead(d *schema.ResourceData, meta interface{
 		}
 	}
 
-	return tags.FlattenAndSet(d, resp.Tags)
+	// TODO: support Tags when this bug is fixed:
+	// https://github.com/Azure/azure-rest-api-specs/issues/5249
```

Comment:
Should we add this to a deprecation message so the user knows why their tags aren't being set?